src/include/execution/ast/builtins.h

```diff
@@ -7,6 +7,16 @@ namespace terrier::execution::ast {
 // The list of all builtin functions
 // Args: internal name, function name
 #define BUILTINS_LIST(F)                                                \
+  /* SQL NULL. */                                                       \
+  F(IsNull, isNull)                                                     \
```

Comment:
Rename `isNull` -> `isSqlNull` to distinguish between pointer nulls and SQL nulls.

Reply:
Renamed.